sql/promscale--0.0.0--0.5.0.sql

```diff
@@ -0,0 +1 @@
+promscale--0.5.0.sql
```

Comment:
where in the code is this symlink created?

Reply:
It isn't... Yet. This has been the thorn in my side for the last couple of days.